src/chains.ts

```diff
@@ -9,15 +9,43 @@ const defaultL2Addresses = {
   fallbackHandlerAddress: "0x1AC114C2099aFAf5261731655Dc6c306bFcd4Dbd",
   createCallAddress: "0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4",
   signMessageLibAddress: "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
-  safeSingletonAddress: "0x",
+  safeSingletonAddress: "0x3E5c63644E683549055b9Be8653de26E0B4CD36E",
   simulateTxAccessorAddress: "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
 };
 
 // Example how to add new networks before they are merged and released from `safe-global/safe-deployments` package.
 export const contractNetworks: ContractNetworksConfig = {
   [`${chains.zoraSepolia.id}`]: defaultL2Addresses,
+  [`${chains.blastSepolia.id}`]: defaultL2Addresses,
+  [`${chains.optimismSepolia.id}`]: defaultL2Addresses,
 };
 
+// const blastSepolia = {
```

Comment:
can this be deleted?

Reply:
I want to leave it for when blast mainnet is out it's a useful template because they won't be in the viem package.